simple_template_server/README.md

```diff
@@ -0,0 +1,50 @@
+Template Development Server
+===================================
+
+This is a simple Snippet Server for development of Snippet Templates. It ignores
+the client configuration including locale, Firefox version and release channels
+and just returns all the snippets to the requesting client. If you only care for
+Snippet Template development this is the server for you.
+
+Installation
+------------
+
+1. Create and activate a Python virtualenv
+
+    ``` shell
+    virtualenv venv
+    . venv/bin/activate
+    ```
+
+2. Install the needed packages
+
+    ``` shell
+    pip install -r requirements.txt
```

Comment:
I think the:

```
(venv)[scott@localhost snippets-service]$ pip install -r requirements.txt
You are using pip version 6.0.8, however version 8.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

```

Is unrelated though? Possible to set version 8.1.1 in the venv? Once I do `pip install --upgrade pip` that error goes away. I'm also able to remove the previous mysql error with installing mysql-deval.